### DISCUSSION TITLE
QDMR 0.10.2 - can not add private call to RxG

### DISCUSSION THREAD

**Discussion by thunderstar5000** [hmatuschek/qdmr#219] [Ideas]
QDMR 0.10.2: When creating a new RxGroup, you can only add Talkgroups, but no private calls. This is only possible in Opengd77CPS.

Did I miss something?
Best regards
Hans

**Comment by hmatuschek** [comment]
Actually, this is intentionally as the majority of the CPS only allow to add group calls to the group lists.

**Comment by thunderstar5000** [comment]
So I did not missing something. Maybe you might want add this option in one of the next versions? Generally this feature is technically possible, as the Opengd77CPS is offering this.

Best regards

Hans

**Comment by hmatuschek** [comment]
I've turned this issue in a discussion. I'll check if also other radios handle private calls in group lists. I still don't understand how the radio should behave if a private call is in the group list. Private calls to your ID are always received irrespective of the current group list. So does this feature allows for listening in on private calls to those in the group list?

**Comment by thunderstar5000** [comment]
The opengd77 firmware allows a special structure. Unlike the other firmwares, where you name a zone to a repeaters call sign, then add one channel in it and several TGs with its own TS, opengd77 allows another structure: 

Create a zone and name it as you want i.e. "homezone". In this homezone put several repeaters and link each to one or several RxG.

Doing so, you scroll with up / down through channels, and with left / right through TGs. This is super comfortable and additionally it saves lots of memory.

But the disadvantages are two. The minor one is, that if the same TG on different repeaters got different TS, then it will not automatically change the TS. Ok, this is mostly not needed. Here on our repeater we got 2 clusters on TG8, one with TS1, the other with TS2. Changing fromn Cluster 1 (TG8 / TS1) to another repeater with same TG8, would not automatically change from TS1 to TS2 (as the old structure does). But that only far aside.

The more important thing is, you can not add any private calls into an RxG, so it is not possible to scroll through the TGs (left and right), and chose a private call. This includes of course any personal callsign, the "parrot" and "info" TG.

Of course maybe I could trick the CPS, and set any privatecall, parrot and info as a seperate TG, but maybe this might crash other things.

However, this may touch only opengd77 and no other devices. And it is not a bug, more a feature, which might be implemented in future versions.

Best regards

Hans

**Comment by hmatuschek** [comment]
Just to mention it here, the upcoming release 0.10.3 will include this feature. In fact, you can add private calls to group lists since 0.10.0, however, not using the GUI qdmr but by hand within the yaml file (this was actually a bug).

**Comment by thunderstar5000** [comment]
Am Mittwoch, 10. August 2022, 10:11:02 CEST schrieb Hannes Matuschek:
Hi Hannes,
looks good! I will wait some time and recheck the last version. In the 
meantime I compiled the last code and found no issues so far.

However, I still had no time to test everything, but I will keep you informed.

Thank you very much for your work.

Issue #219 has also to be rechecked by me, I will do it the next few days and 
then sending you the result.

Best regards and 73s 

Hans, DL4OCJ


> Just to mention it here, the upcoming release 0.10.3 will include this
> feature. In fact, you can add private calls to group lists since 0.10.0,
> however, not using the GUI qdmr but by hand within the yaml file (this was
> actually a bug).
